features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/TimelinePresenter.kt

```diff
@@ -81,6 +82,7 @@ class TimelinePresenter @AssistedInject constructor(
             computeReactions = true,
         )
     )
+    private val timelineItems = mutableStateOf<ImmutableList<TimelineItem>>(persistentListOf())
```

Comment:
You can probably use `private var timelineItems by` and it should work, and you won't need the `var timelineItems by this.timelineItems` line below.

Reply:
Yes you're right, I don't know why on my mind it was only possible inside `Composable` 😅